make/mt-config/tests/mt-native

```diff
@@ -6,3 +6,4 @@
 #XI_RUN_UTESTS = $(XI_UTESTS) -l0 --terse
 XI_RUN_UTESTS := (cd $(dir $(XI_UTESTS)) && LD_LIBRARY_PATH=$(dir $(XI)):$$LD_LIBRARY_PATH exec $(XI_UTESTS) -l0)
 XI_RUN_ITESTS := (cd $(dir $(XI_ITESTS)) && LD_LIBRARY_PATH=$(dir $(XI)):$$LD_LIBRARY_PATH exec $(XI_ITESTS))
+XI_RUN_FTEST = (cd $(XI_FUZZ_TESTS_BINDIR) && $(1) $(XI_FUZZ_TESTS_CORPUS_DIR)/$(notdir $(1))/ -max_total_time=10 -max_len=256); 
```

Comment:
I was more thinking of differentiating between local and Travis's builds. So locally you can leave tests for longer and occupy use more CPU cores. Travis however requires some gentle treatment - so that's why I was thinking about introducing parameters responsible for time and parallelization etc.